apps/examples/testcase/le_tc/kernel/tc_pthread.c

```diff
@@ -626,6 +636,7 @@ static void tc_pthread_pthread_create_exit_join(void)
 {
 	int ret_chk;
 	pthread_t pthread;
+	int task_pid;
```

Comment:
how about to initialize the task_pid?

Reply:
Thanks for review. But I think we don't need to initialize this variable.
Because before using this variable, it always set the value in line 658.